Berksfile.lock

```diff
@@ -158,20 +157,22 @@ GRAPH
     systemd (= 2.1.3)
     zabbix (= 0.0.43)
     zabbix-custom-checks (>= 0.0.0)
-  ufw (3.2.0)
+  ufw (3.2.1)
     firewall (>= 2.0)
   user (0.7.0)
   users (2.0.3)
-  windows (5.0.0)
+  windows (5.2.3)
   xfs (2.0.1)
   yum (5.1.0)
-  yum-epel (3.2.0)
+  yum-epel (3.3.0)
   zabbix (0.0.43)
     apache2 (>= 1.0.8)
     database (>= 1.3.0)
     mysql (>= 1.3.0)
     postgresql (>= 0.0.0)
     ufw (>= 0.6.1)
     yum (>= 0.0.0)
-  zabbix-custom-checks (0.2.9)
+  zabbix-custom-checks (0.2.14)
+    seven_zip (< 3.0.0)
```

Comment:
Downgrade for seven_zip is on purpose?

Reply:
Not on purpose, but I don't care since we don't need it. zabbix-custom-checks/metadata.rb says:
```
# For compatibility with Chef 12.x
depends          "seven_zip",    "< 3.0.0"
```

So let's leave this untouched.